test/testfixture/make_functions.go

```diff
@@ -515,3 +516,35 @@ func makeTrackers(fxt *TestFixture) error {
 	}
 	return nil
 }
+
+func makeQueries(fxt *TestFixture) error {
+	if fxt.info[kindQueries] == nil {
+		return nil
+	}
+	fxt.Queries = make([]*query.Query, fxt.info[kindQueries].numInstances)
+	queryRrepo := query.NewQueryRepository(fxt.db)
+
+	for i := range fxt.Queries {
+		fxt.Queries[i] = &query.Query{
+			Title:   testsupport.CreateRandomValidTestName("query "),
+			Fields:  `{"space": "a2bda47f-4c4a-400c-9e30-62b11eaf8900"}`,
```

Comment:
This needs to go away I'd say. Can you initialize with no value for `Fields` and do something like this instead?

```go
if !fxt.IsolatedCreation {
    fxt.Queries[i].Fields= fmt.Sprintf(`{"space": "%s"}`, fxt.Spaces[0].ID)
    fxt.Queries[i].SpaceID = fxt.Spaces[0].ID
}
```

Reply:
yes done in https://github.com/fabric8-services/fabric8-wit/pull/1824/commits/354050c56766743806b6eaeb78751e72c7546d95